### PR TITLE
Plastic Surgery 2.0

### PR DESCRIPTION
**Pull request by dragoslava**
Originaly I decided to just port from paracode but decided to just code it a more simple way with less hassle of porting it.

"Added the ability to do plastic surgery to fix disfigured face's and for other purposes. Credit to MarinaGryphon for Assistance"

dreamseeker_2018-10-20_20-43-06

### CODE REVIEW

**Comment by dragoslava** [comment]
!feedback

**Comment by dragoslava** [comment]
!review

**Comment by furrycactus** [comment]
There's already facial reconstruction surgery to fix disfigured faces. What other things would this add out of curiosity?

File: code/modules/surgery/plastic_surgery.dm [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+		"You start to alter [target]'s face and neck with \the [tool].")
+		..()
+
+	end_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		var/obj/item/organ/external/head/head = target.get_organ(target_zone)
+		if(head.disfigured)
+			head.disfigured = FALSE
+			user.visible_message("[user] successfully restores [target]'s appearance!", "<span class='notice'>You successfully restore [target]'s appearance.</span>")
+
+		var/getName = sanitize(input(user, "What is your patients new identity?", "Name change") as null|text, MAX_NAME_LEN)
```
Comment (HeyBanditoz) [inline code/modules/surgery/plastic_surgery.dm]: Patient's

**Pull request merged**